pkg/loop/internal/config.go

```diff
@@ -62,7 +62,7 @@ func (o *offchainConfigDigesterClient) ConfigDigest(config libocr.ContractConfig
 }
 
 func (o *offchainConfigDigesterClient) ConfigDigestPrefix() (libocr.ConfigDigestPrefix, error) {
-	ctx, cancel := o.ctx()
+	ctx, cancel := o.ctxAndCancelFromStopCh()
```

Comment:
Is this name change just for readability? It was introduced to be a shorthand form, but with a long name like this it doesn't save much over in-lining:
```suggestion
	ctx, cancel := utils.ContextFromChan(b.StopCh)
```
If you think is important info to surface at callsites for readers, then I would favor inlining which seems easier to read.